.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: circleci/ruby:2.5.1-node-browsers
+    steps:
+      - checkout
+      - run:
+          name: Ruby dependencies
```

Comment:
FYI circleci ha un meccanismo di cache delle dipendenze, velocizza notevolmente la build: https://circleci.com/docs/2.0/caching/

Reply:
Sì grazie, lo conosco!
In un [mio fork](https://github.com/francescozaia/designers.italia.it/blob/master/.circleci/config.yml) l'avevo anche testato, per ora vista l'urgenza volevo solo assicurarmi che il resto fosse ok e poi lo sistemavo.
Tale meccanismo andrebbe infatti inserito anche su [developers](https://github.com/italia/developers.italia.it/blob/master/circle.yml).